### PR TITLE
[Merged by Bors] - feat: forward-port 19107

### PR DESCRIPTION
**Pull request by urkud**
Forward-port leanprover-community/mathlib#19107

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by ADedecker** [comment]
I'm going to open a PR in mathlib3 golfing the `arrow_congrSL` construction, would it be better for this to wait? Or will we need 2 PRs anyway?

**Comment by urkud** [comment]
Are you golfing proofs? Then why not do it directly in mathlib 4?

**Comment by urkud** [comment]
@fpvandoorn I had to change `simps` config for `ContinuousLinearEquiv`. Do we need to change config for other equivs as well?

File: Mathlib/Topology/VectorBundle/Basic.lean [hunk 1]
```diff
@@ -60,7 +60,7 @@ Vector bundle
 
 noncomputable section
 
-open Bundle Set Classical
+open Bundle Set Classical Topology
```
Comment (eric-wieser) [inline Mathlib/Topology/VectorBundle/Basic.lean:63]: This line isn't in on the port-status page.

Comment (eric-wieser) [reply Mathlib/Topology/VectorBundle/Basic.lean:63]: If it's needed
```suggestion
open scoped Bundle Set Classical Topology
```

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

**Comment by eric-wieser** [comment]
> I'm going to open a PR in mathlib3 golfing the `arrow_congrSL` construction, would it be better for this to wait? Or will we need 2 PRs anyway?

It's easier to review forward-ports one commit at a time, unless your refactor is so big that you plan to re-port the entire file from scratch.

**Comment by ADedecker** [comment]
> Are you golfing proofs? Then why not do it directly in mathlib 4?

Not only, I'm also touching statements a bit and adding general versions (the PR is leanprover-community/mathlib#19128).

**Comment by ADedecker** [comment]
> > I'm going to open a PR in mathlib3 golfing the `arrow_congrSL` construction, would it be better for this to wait? Or will we need 2 PRs anyway?
> 
> It's easier to review forward-ports one commit at a time, unless your refactor is so big that you plan to re-port the entire file from scratch.

Ok, let's do two porting PRs then!

File: Mathlib/Topology/VectorBundle/Basic.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Topology/VectorBundle/Basic.lean:63]: I need it for the `Continuous[_, _]` notation. IMHO, it's more readable than `@Continuous _ _ _ _ f`

```suggestion
open Bundle Set Classical
open scoped Topology
```

**Comment by urkud** [comment]
@ADedecker My goal here was to reduce the "out-of-sync" queue so that I can merge some of my mathlib3 PRs.

**Comment by ADedecker** [comment]
Well leanprover-community/mathlib#19128 won't get merged before we go under the threshold anyway, so you can probably merge your PRs first 😅
I'm also totally fine with doing these changes by hand in mathlib4, but I think that means more work for the reviewer to check the translation is accurate.

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**